source/publishing_data/07_configuring_visualizations/01_configuring_table_view/table.rst

```diff
@@ -1,22 +1,46 @@
-Configuring the table view
-==========================
+Configuring the Table visualization
+===================================
 
-.. admonition:: Note
-   :class: note
+.. admonition:: Prerequisite
+   :class: important
 
-   The table view is always available on all non-empty datasets.
+   For the Table visualization to be available, the dataset must not be empty.
 
-You can configure which fields are displayed, and in which order. Note that it
-only changes the table visualization; removing a field from the table view will
-not remove it from the dataset itself, and it will still be available in other
-visualizations and if you export the dataset.
 
-.. ifconfig:: language == 'en'
+In the edition interface of a dataset, the Visualizations > Table tab allows to configure the Table visualization displayed in the front office, once the dataset is published on the portal.
 
-    .. image:: images/table__table--en.jpg
-        :alt: Table view
+The Table visualization, when available, cannot be disabled.
 
-.. ifconfig:: language == 'fr'
+.. screenshot
 
-    .. image:: images/table__table--fr.jpg
-        :alt: Vue Table
+To configure the Table visualization, follow the indications from the table below.
+
+.. list-table::
+  :header-rows: 1
+
+  * * Option name
+    * Description
+    * Usage
+  * * Default sort
+    * By default, the Table visualization keeps the order of the records as it is after they been processed. It is possible to define another default sort, by choosing a field and a sort type.
```

Comment:
after the **have** been processed

Reply:
@thoumasd @richterb It seems incorrect to me, the order without the `sort` option is not the one of the publication, it's random or undefined isn't it ?

Reply:
@remilejeune you're right, I didn't pay attention to this one, the default sort is undefined indeed.